javascript/javascript1/pizza-exercise/pizzaNew.js

```diff
@@ -0,0 +1,16 @@
+const amountOfPizza = prompt("How many pizaas you want to order?");
+const familySizePizza = confirm("Do you like to order family size pizza ? if yes click ok else click cancel : ");
+const pizzaPrice = 60;
+const nameOfPizza = prompt("Enter the name of the pizza ?");
```

Comment:
Good thinking with using `prompt` and `confirm` for these. Makes the application more interactive.

Reply:
There's unfortunately no way to do this. 
The alert boxes are "native" to the browser, so you can't really change the labels.
If you want you can make a prompt asking for the user to answer "Yes" or "No"